src/pages/version/VersionTabs.tsx

```diff
@@ -86,9 +86,9 @@ export const VersionTabs: React.FC<Props> = ({
   isPatch,
   taskCount,
 }) => {
-  const { id, tab } = useParams<{ id: string; tab: PatchTab }>();
+  const { [slugs.versionId]: versionId, [slugs.tab]: tab } = useParams();
```

Comment:
we lose some type specificity without `tab: PatchTab`; maybe want to put it back?